sdk/release_notes.md

```diff
@@ -6,13 +6,12 @@
 
 ### Microsoft.Azure.Functions.Worker.Sdk <version> (meta package)
 
-- <entry>
+- Default to source-generated function metadata (#1968)
 
```

Comment:
Should we also mention something like "If you had `<FunctionsEnableWorkerIndexing>True</FunctionsEnableWorkerIndexing>` in your csproj, you can remove it after upgrading to this version"  in the release notes when we release this version of SDK ?

Reply:
Do you plan to include the version(`MinorProductVersion`) bump in a this PR itself?

Reply:
@kshyju I will add that comment. I will include the version bump in the release prep PR for whatever package ends up having this change.